src/group/components/GroupEdit.vue

```diff
@@ -97,12 +123,70 @@
           </template>
         </QSelect>
 
+        <h4 class="text-primary q-mt-xl q-mb-lg">
+          {{ $t('GROUP.HEADINGS.NEW_MEMBERS_SIGNUP') }}
+          <QBtn
+            size="sm"
+            round
+            flat
+            icon="fas fa-info-circle"
+            @click.prevent
+          >
```

Comment:
I think we have 3 of these "more info" dialogs now! Which is the magic number for extracting into it's own component :) Not required, but if you felt like doing it then could be nice.

Reply:
yep, we'll do ;)